src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1249,7 +1258,7 @@ def visit_span(self, span: Span):
                     fingerprint=fingerprint,
                     op=span.get("op"),
                     desc=span.get("description", ""),
-                    parent_span_ids=[],
+                    parent_span_ids=[span.get("parent_span_id")],
```

Comment:
Hold on, `parent_span_ids` here refers to something else, not the actual parent of the current span. The parent span refers to the span that is the parent of the repeating / offender spans. I believe this gets populated elsewhere, and setting it here might break things.

Reply:
As far as I can tell its only used in the presentation layer:
![image](https://user-images.githubusercontent.com/4205004/204663317-df94b728-975a-4d64-8657-9c76ed253ab3.png)

Reply:
Sorry, I think my brain is dead right now, I completely misread the code here in the collapsed unified view and thought this was under the N+1 detector 😂 . It would have no effect on fingerprinting anyways as you mentioned.